src/bp/admin/ui/src/auth/Login.tsx

```diff
@@ -78,7 +78,7 @@ const Login: FC<Props> = props => {
   const loadAuthConfig = async () => {
     const { data } = await api.getAnonymous().get('/admin/auth/config')
```

Comment:
Can you add typings for this response! (and also on line 86 for `strategyId` at the same time?)